Broken makefile because of RHEL_RELEASE_CODE

I can no longer build tracee-ebpf with the makefile

```
fedo grant in ~> tracee-ebpf on ~> staticbuild
[*] make                                                                                130 ↵
/bin/sh: << 8 | : syntax error: operand expected (error token is "<< 8 | ")
clang \
	-I dist/libbpf/usr/include/bpf \
	-I 3rdparty/include \
	-I tracee/x86_64 \
	-DCORE \
	-D__TARGET_ARCH_x86 \
	-D__BPF_TRACING__ \
	-D__VM_LINUX_H__ \
	-D__KERNEL__ \
	-DLINUX_VERSION_CODE=331014 \
	-DRHEL_RELEASE_CODE= \
	-target bpf \
	-O2 -c -g -o dist/tracee.bpf.core.o tracee/tracee.bpf.c
tracee/tracee.bpf.c:323:24: error: invalid token at start of a preprocessor expression
#if (RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(8, 0))
                       ^
1 error generated.
make: *** [Makefile:129: dist/tracee.bpf.core.o] Error 1

```

The content of my /etc/redhat-release file is:

```
[*] cat /etc/redhat-release                                                              
Fedora release 33 (Thirty Three)
```

uname
```
[*] uname -a                                                                            130 ↵
Linux localhost.localdomain 5.13.6-100.fc33.x86_64 #1 SMP Wed Jul 28 15:30:10 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

--- a/tracee-ebpf/Makefile
+++ b/tracee-ebpf/Makefile
@@ -39,9 +39,12 @@ DOCKER_BUILDER_KERN_SRC ?= $(if $(shell readlink $(KERN_SRC_PATH)),$(shell readl
 DOCKER_BUILDER_KERN_SRC_MNT ?= $(dir $(DOCKER_BUILDER_KERN_SRC))
 LINUX_VERSION_CODE := $(shell uname -r | awk '{split($$0,a,"."); split(a[3], b, "-"); print lshift(a[1], 16) + lshift(a[2],8) + b[1];}')
 
-ifneq (,$(wildcard /etc/redhat-release))
-	RHEL_MAJOR := $(shell grep -Eoh -m 1 '[0-9]+\.[0-9]+' /etc/redhat-release | cut -d '.' -f 1)
-	RHEL_MINOR := $(shell grep -Eoh -m 1 '[0-9]+\.[0-9]+' /etc/redhat-release | cut -d '.' -f 2)
+DIST_ID := $(shell grep -m 1 'ID=' /usr/lib/os-release | cut -d '=' -f '2' | sed 's/\"//g')
+VERSION_ID := $(shell grep -m 1 'VERSION_ID=' /usr/lib/os-release | cut -d '=' -f '2' | sed 's/\"//g')
+CENTOS_MERGED_RHEL := $(shell echo $(DIST_ID) | sed 's/centos/rhel/')
+ifeq ($(CENTOS_MERGED_RHEL), rhel)
+	RHEL_MAJOR := $(shell echo $(VERSION_ID) | cut -d '.' -f 1)
+	RHEL_MAJOR := $(shell echo $(VERSION_ID) | cut -d '.' -f 2)
 	RHEL_RELEASE_CODE := $(shell echo $$(($(RHEL_MAJOR) << 8 | $(RHEL_MINOR))))
 else
 	RHEL_RELEASE_CODE := 0

--- a/tracee-ebpf/Makefile
+++ b/tracee-ebpf/Makefile
@@ -44,7 +44,7 @@ VERSION_ID := $(shell grep -m 1 'VERSION_ID=' /usr/lib/os-release | cut -d '=' -
 CENTOS_MERGED_RHEL := $(shell echo $(DIST_ID) | sed 's/centos/rhel/')
 ifeq ($(CENTOS_MERGED_RHEL), rhel)
 	RHEL_MAJOR := $(shell echo $(VERSION_ID) | cut -d '.' -f 1)
-	RHEL_MAJOR := $(shell echo $(VERSION_ID) | cut -d '.' -f 2)
+	RHEL_MINOR := $(shell echo $(VERSION_ID) | cut -d '.' -f 2)
 	RHEL_RELEASE_CODE := $(shell echo $$(($(RHEL_MAJOR) << 8 | $(RHEL_MINOR))))
 else
 	RHEL_RELEASE_CODE := 0

--- a/tracee-ebpf/Makefile
+++ b/tracee-ebpf/Makefile
@@ -26,6 +26,8 @@ BPF_BUNDLE := $(OUT_DIR)/tracee.bpf.tar.gz
 LIBBPF_SRC := 3rdparty/libbpf/src
 LIBBPF_HEADERS := $(OUT_DIR)/libbpf/usr/include
 LIBBPF_OBJ := $(OUT_DIR)/libbpf/libbpf.a
+OS_RELEASE_FILE := /etc/os-release
+
 # static build:
 ifdef STATIC
 CGO_EXT_LDFLAGS += -static
@@ -39,8 +41,8 @@ DOCKER_BUILDER_KERN_SRC ?= $(if $(shell readlink $(KERN_SRC_PATH)),$(shell readl
 DOCKER_BUILDER_KERN_SRC_MNT ?= $(dir $(DOCKER_BUILDER_KERN_SRC))
 LINUX_VERSION_CODE := $(shell uname -r | awk '{split($$0,a,"."); split(a[3], b, "-"); print lshift(a[1], 16) + lshift(a[2],8) + b[1];}')
 
-DIST_ID := $(shell grep -m 1 'ID=' /usr/lib/os-release | cut -d '=' -f '2' | sed 's/\"//g')
-VERSION_ID := $(shell grep -m 1 'VERSION_ID=' /usr/lib/os-release | cut -d '=' -f '2' | sed 's/\"//g')
+DIST_ID := $(shell grep -m 1 'ID=' $(OS_RELEASE_FILE) | cut -d '=' -f '2' | sed 's/\"//g')
+VERSION_ID := $(shell grep -m 1 'VERSION_ID=' $(OS_RELEASE_FILE) | cut -d '=' -f '2' | sed 's/\"//g')
 CENTOS_MERGED_RHEL := $(shell echo $(DIST_ID) | sed 's/centos/rhel/')
 ifeq ($(CENTOS_MERGED_RHEL), rhel)
 	RHEL_MAJOR := $(shell echo $(VERSION_ID) | cut -d '.' -f 1)

